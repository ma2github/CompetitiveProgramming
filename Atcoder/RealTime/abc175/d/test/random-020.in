898 542441312
894 845 520 17 828 348 261 503 681 798 419 788 481 241 364 145 263 839 538 176 352 592 231 814 337 382 75 272 327 457 60 622 665 806 259 790 497 445 856 242 248 881 717 82 598 796 559 649 511 797 536 310 36 296 652 13 53 437 699 167 369 388 760 384 29 678 247 443 749 783 868 540 381 766 463 195 328 135 146 585 48 119 657 217 99 833 566 275 106 178 719 488 83 600 631 130 608 386 807 883 298 107 232 748 470 357 160 204 889 376 656 449 229 627 2 33 427 580 711 409 565 230 336 133 629 23 769 96 341 267 505 680 785 161 465 858 31 692 361 429 892 183 866 322 402 452 128 809 720 826 730 855 684 139 343 117 789 615 215 660 530 8 659 726 746 120 426 7 697 455 155 840 377 689 206 43 815 170 300 165 72 28 654 26 570 548 173 813 18 81 393 354 397 307 879 260 251 482 539 408 853 671 265 661 297 887 568 632 878 823 68 485 487 10 273 676 391 466 355 225 895 468 819 250 618 675 339 389 515 228 606 804 623 896 668 330 888 197 287 190 439 779 282 172 151 837 642 885 108 682 584 525 460 803 561 724 510 431 100 795 589 898 552 338 370 500 235 831 640 569 644 664 252 63 185 271 574 502 375 345 519 434 841 830 413 616 824 567 353 857 220 602 775 890 166 685 549 394 563 52 635 758 670 270 558 601 588 528 184 484 690 792 249 20 30 822 581 387 111 378 453 582 713 372 87 317 537 555 380 56 479 131 395 734 820 61 289 695 450 89 302 605 38 634 743 522 733 735 708 34 619 227 542 234 379 211 350 84 373 464 320 279 524 118 571 728 794 867 756 333 432 441 91 742 262 233 817 430 244 475 92 842 143 80 770 739 729 780 415 311 125 16 224 5 738 686 688 180 490 512 19 268 641 9 886 836 696 876 740 736 458 744 709 405 846 721 835 747 276 331 58 136 737 202 148 554 607 850 718 191 219 508 870 531 164 786 294 109 543 383 865 66 365 764 651 679 440 507 573 604 546 691 73 544 448 517 702 116 254 715 142 480 54 174 129 461 308 340 50 329 587 255 212 102 266 256 288 694 169 722 535 800 319 628 187 95 237 205 358 754 683 321 636 313 666 407 404 752 189 214 710 862 152 134 778 637 533 349 658 290 612 456 774 418 359 489 210 216 594 126 491 62 787 447 37 854 818 366 258 825 385 469 147 513 121 639 368 344 124 196 153 274 157 198 188 201 67 516 374 292 849 428 11 332 576 347 677 871 149 811 406 501 399 669 304 609 35 474 141 278 714 757 442 610 363 112 390 526 97 579 848 745 476 693 534 326 168 725 334 626 154 351 625 731 808 765 79 514 15 802 621 257 433 560 776 314 4 425 701 645 821 791 59 494 812 182 462 805 123 893 700 454 1 421 596 400 316 162 613 312 200 47 504 71 884 782 572 897 763 192 243 759 838 114 398 51 435 716 115 732 727 591 614 772 44 873 498 42 557 799 323 633 829 647 55 181 495 578 284 477 882 223 577 562 127 113 86 451 39 773 303 226 541 301 158 85 77 411 655 360 371 767 424 595 545 472 707 88 76 140 315 529 674 864 834 286 306 762 417 436 753 423 860 444 875 238 137 177 14 6 706 630 21 414 179 506 93 222 877 122 438 771 416 761 863 492 24 396 523 663 98 22 816 611 486 624 599 617 245 704 342 852 672 648 527 698 847 874 277 499 27 281 74 471 104 285 420 236 793 253 356 199 291 194 193 777 810 446 586 218 781 861 851 412 40 556 646 493 653 673 564 213 90 620 723 832 32 41 667 643 551 880 78 110 57 392 346 335 827 650 768 583 69 171 638 859 869 403 801 239 891 305 324 784 209 208 687 138 459 532 518 843 280 705 283 101 45 662 575 64 603 25 150 299 496 509 590 12 318 94 478 422 65 103 264 521 144 295 175 240 401 159 156 844 132 593 703 49 750 553 293 3 203 207 325 597 872 751 70 741 186 483 712 246 309 105 755 362 221 367 547 473 163 410 46 550 269 467
-107651062 -612508856 -758180747 16699117 831720571 125728460 -787278143 -621609942 764051285 -797988229 474357266 -301917581 364592986 -737795913 682697869 -828251160 687957374 -482381807 -48659153 -411691939 49395156 987937200 -441554191 53536791 592647997 554415358 -161696282 -504590060 216343201 587257371 -495791173 148485264 -99110594 -371284547 -814188173 941195775 160457740 970350693 -418688417 -674115589 130118819 937831815 68172313 -533341017 365002280 695334869 955082592 -289870262 406035135 -930696917 -232797059 929877736 758493881 -832452943 -546179963 526594632 -815242748 -596464493 99279650 -1478285 554842558 764915474 272487639 456656914 725410753 -490882081 160496365 974046048 -449974997 -739946549 -394564040 -290905850 -930302089 347670077 -297803699 996218165 490679074 -18528585 -299215989 36544160 -324607175 811877857 70541482 -86976027 712664335 -785386988 919050497 -564826379 -6096094 -978451471 -223709274 729715344 948578440 -31431641 -281644580 189755230 -933017404 362325049 281484981 284235988 -400733838 -896724423 -838487367 843504633 -166790245 -581150521 877071706 490863694 431781219 767653265 873730635 -195744677 732862308 871599064 962101027 -618371519 279619530 -595225656 662649622 16624024 499254982 -843828599 695432526 -853407381 221440294 796153501 184870214 599392385 -936428556 -621750803 -376858368 572363642 628065586 81462380 125275344 256369026 -428532115 -302035424 734851139 481231783 194031889 588514802 -781427802 -808286341 727626255 761616573 229703679 -251042646 617638881 775270685 908531671 336064987 -241371576 -350480024 907012438 803197644 -197985225 -591011484 -279202809 601585475 -659718513 -898335894 -237016320 51120211 -715631601 986785366 -929664422 -596093562 563222585 -992162337 -571745671 -905878905 889727338 611009899 396028191 123177264 -951339502 -625925130 -701999826 -128805127 353967492 578837762 819629754 131016934 -204104946 -905748897 14875405 271089480 -351891687 -109431446 869884395 -380078635 157784332 -564711966 156133536 500495140 374733487 -872757526 -940351878 788924151 58842134 -539912781 767652620 561597335 -636889007 729239534 -820777094 -602464378 476317489 452933170 -538107132 -538906202 703759744 988932360 375452970 784229111 612005242 -217567969 -851950624 -667512693 745644463 -857695044 -453259520 882263995 200741624 -669793880 -760197106 872634045 966356148 217168651 -96369488 577860315 -104493172 -76061316 -598695031 -687514286 -51824671 287824340 -176731461 -972624803 -695459433 -203373914 657189699 -916924798 -397087462 422847742 166915165 -251811165 182536648 663854813 722744765 994152772 -988418683 -708249431 883330986 -234077283 240456953 940653664 835877434 923110896 959915925 -543198506 -435255232 -239478754 -306946521 573836480 -945790225 -833910117 603405930 906811966 -622660947 52988280 -202329533 296327706 720947515 679201925 481942950 -481642734 178545133 902813702 144011387 479014699 683044382 -735014269 625205366 -168496145 -241689636 283147191 363798941 -558285976 -240490905 42803662 -38609566 290815374 -372996749 -52678841 -589386050 -283042067 -891897020 839984446 -624069475 -784780924 -537102365 -302218096 -358023425 322311645 871027582 -597585576 32863711 251368445 665876595 458889148 -774079024 859734340 935014074 -200916420 -328180749 -978054356 708653372 448921651 987376459 930825320 -467751506 840723191 -392406340 944336587 -228942535 -71239869 173372358 752255998 218446695 962265500 302969659 -911101542 -51947386 526644549 137198572 667443472 -836062531 -170434409 -25092501 324458937 -315415478 -459993653 404310841 61134928 -907021473 -520485992 -11319141 42200392 -740017921 179642307 83002835 -355992312 -37650315 347938323 -65683413 -198523671 -823334020 -75097559 -678118016 549418004 727444811 370334736 404374614 5626366 187643165 885734424 691276964 -382507665 -814539554 -940216109 -992259874 -370128193 693243748 -380265509 113429057 514535699 -222562121 -476979051 -731103191 521456532 356317598 -270456082 124430285 238105348 847779556 -107748336 -566156293 103342825 549098380 -878297700 -29528718 -742052313 853729370 310567850 417616083 -516892042 -671417629 109533602 939008715 705747302 882885034 433248759 -454210456 -36134025 368720834 988815271 534589471 -329006604 -473427271 333409641 -240421130 -726821425 998094281 -983787794 -297101204 -57437264 -550644535 186121403 -585872582 -828065521 198229737 -627800119 -662307501 -796815788 107497636 415186755 246446482 -541713957 787412809 -942122398 -394118584 -646646494 687901921 -142897086 125966822 124009006 -147903242 405782610 588662367 -121740461 -937894162 -85325501 -21935490 554301618 -247213761 -750003257 381200979 -24302135 -713888963 906707102 247492438 -715254708 -790598903 -162930894 -945921948 212086197 853799858 -690508790 564931144 -41764938 917891578 -65944968 -193720303 770028054 835404544 689203829 -870732859 242640180 270023435 -132259121 -895789904 -784512385 -165427271 -881186515 -467990149 -936665733 859923993 204280003 -732648278 756388301 -615141721 901552873 -636077342 946506482 -770699846 589150551 -323810932 446473015 -745306173 -434320824 445406592 50822964 -471452279 955895683 242472689 323769917 987098955 -795860492 -926033959 -731490112 454594600 813687476 -837055344 722301848 -764200076 -961823537 -340895804 -163027470 -341308232 79508513 -113965100 58684422 -877705531 -292540702 655658696 886935569 -810866285 -952349985 -339299635 233858022 -834491701 -353511190 677104953 -171666962 -312963821 -682326876 -985994674 861578788 -204037245 -770685284 -818931270 -304299051 -904541161 632536692 -47211390 -910437379 -637257649 -818931218 462066180 529640192 -122307787 -860172369 117044927 491963094 -301850683 -102093836 204085482 248800155 -127018200 749792138 464343642 -378196642 213864057 -771427858 -679232637 -173386448 585092486 752417028 59963944 -78808497 -814850752 -177392148 610881670 826991453 -652209423 433067309 511412772 -821742962 -674206830 818211924 851647864 50382299 495852289 166164180 559627412 -659706178 25711370 402838887 -23076493 863184322 -26010955 516390190 18941815 -396984762 -446408718 -354099531 -615071671 -924957194 -944272495 -708869785 -973606937 -248454182 -342882035 -983967558 225013736 325179365 -701398588 -57063419 -197234098 476441775 530526992 -740641474 -80055908 -824803050 742603113 530649118 -185698716 178693151 908593575 242914326 -908693519 -372535182 -906123660 -329814904 -183791330 915872179 -784536057 -132433595 -719216413 -826951688 -649897777 158801325 740028768 13717069 196179917 766478007 -325593067 756133815 -858594364 -669580930 -901027386 -140482401 -970662926 -544682545 547882507 -929725623 427744634 -18995308 -554830842 435462268 673534905 -759482828 102636862 -6501825 -85468636 -509505486 462462555 -66210544 389119243 817681771 -401748456 -462355918 -107639853 527789269 -635681650 46667282 -10127815 811530911 750936750 584144780 751500058 -188194345 -254702715 -329526607 579784996 -750574546 -896126022 949140514 -374696715 213935422 271641304 440118452 -458925433 -880112705 68694286 359425130 -211556580 -26257580 -939968921 334892019 172389233 -81867821 11551015 -152611731 -479758162 769156333 435799330 -923584779 262428410 652060623 -188835933 -415967753 -988743829 258056980 544205304 -799381687 -141436710 98242590 763274357 -386077070 -613184339 596328394 -586213655 -773811123 912469922 -627232383 -662235683 955830755 -689392745 -858517041 621325271 318661306 680865316 309315866 329693968 -519325368 -805982956 -459727041 204139 423167383 -215948118 -758680933 -872269953 52202265 131117980 26807489 66263077 -183778093 560121696 221895492 558524206 -708639516 732003263 -85436286 -371665639 851605681 832382190 -700879613 120130912 -533473242 859785375 448340036 35478593 247535719 -600579527 222223186 -282559479 485023814 100379208 -740277069 -147090799 -398756899 685694418 -522697370 -344380244 882625105 -455539102 -748526153 -27843867 -795532890 -699308309 581983613 147759836 -552584722 179318274 -241978245 126851581 169374685 -489765491 530428304 -118871707 43504388 -15977396 590318579 -33369989 989118426 402257765 -913211509 758160094 -995532767 -29234552 -653285569 894324021 -399090808 -584542954 854225604 831297687 -990223612 737148556 976542385 720505971 -679810935 239944142 520713013 841285545 -655990142 142522663 628207328 -916600853 236360848 -851028986 -677674663 567810107 -363540253 897902213 -672065056 -73168591 142975211 -468556612 780974598 -400296489 -701137333 213946801 775175011 966449782 908671137 114639599 -152103961 -869479622 -790849403 305058802 601171220 -77729096 -791880717 -709916544 -423108632 -675893550 -329397880 -511159488 -38914865 -533051540 -958517571 748767230 -381077691 -89238026 273690222 -333891014 120813348 559691778 616514983 382584111 383220148 -844812184 832191283 49056878 722518186 -84249981 -790205392 897720216 -184715453 -715378016 36849115 162912533 817424336 -973550917 -891597813 -272924406 -656503554 -383823062 -980827258 -49859558 660622267 810035995 39384939 -718684291 625496039 570622896 282453811 -946756523 -903071569 713509107 429418734 -63247963 -917622171 301197429 528492906 266778149 -521069480 365547041 295349358 -264567246 -78637366 -553410774 -451132078 559424224 679941037 -623197429 -279775813 -455604824 102017444 322263194 -232096138 -932492142 -555738461 676538987 -504494465
