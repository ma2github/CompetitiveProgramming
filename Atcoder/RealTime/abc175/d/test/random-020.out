411257356095403328
